docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

```diff
@@ -41,6 +41,13 @@ would like to install the packages to `C:/oe_prereqs`.
 ./scripts/install-windows-prereqs.ps1 -InstallPath C:/oe_prereqs -LaunchConfiguration SGX1 -DCAPClientType None
 ```
 
+On Windows 10, the Intel PSW is delivered via Windows update. Running the
+executable installer will fail on Windows 10 machines. To skip PSW installation:
```

Comment:
Since Windows 10 ought to be the dominant case now, it seems like the common case should be covered first.  Line 41 then should be discussed as the exception, for legacy systems, rather than being the starting point for this discussion.  That is, reordering these paragraphs would be more helpful to readers.

Reply:
In that case `SGX1-NoDriver` should be the default.  But I don't know about breaking that behavior yet since I still think the majority of SGX users are still using Windows Server (obviously I could be wrong though).

I think I would rather keep this documentation wording until we confirm what the most common case is. Then we can update the tool defaults and the documentation together.